server/tokens.go

```diff
@@ -136,16 +146,89 @@ func (c *Server) blindedTokenIssuerHandler(w http.ResponseWriter, r *http.Reques
 				Code:    http.StatusInternalServerError,
 			}
 		}
+		response = blindedTokenIssueResponse{proof, signedTokens, signingKey.PublicKey()}
+	}
+	return handlers.RenderContent(r.Context(), response, w, http.StatusOK)
+}
 
-		err = json.NewEncoder(w).Encode(blindedTokenIssueResponse{proof, signedTokens, issuer.SigningKey.PublicKey()})
-		if err != nil {
-			panic(err)
+func (c *Server) blindedTokenRedeemHandlerV3(w http.ResponseWriter, r *http.Request) *handlers.AppError {
+	var response blindedTokenRedeemResponse
+	if issuerType := chi.URLParam(r, "type"); issuerType != "" {
+		issuers, appErr := c.getIssuers(issuerType)
```

Comment:
Do we need to validate issues are the correct type for v3

Reply:
https://github.com/brave-intl/challenge-bypass-server/pull/145/commits/c212d77eac67c1cc0369b89c95e4dea5812fd3fe